diff -p -up gnumeric-1.9.17/configure.in.0000 gnumeric-1.9.17/configure.in
--- gnumeric-1.9.17/configure.in.0000	2009-12-08 02:08:39.000000000 +0100
+++ gnumeric-1.9.17/configure.in	2010-01-19 09:25:32.000000000 +0100
@@ -419,9 +419,12 @@ if test "$try_gda" = true; then
 	PKG_CHECK_MODULES(GDA, [libgda-4.0 >= 4.1.1],
 		[gda_msg=yes],
 		[gda_msg="NO.  libgda problem"])
+	OLD_CFLAGS="$CFLAGS"
+	CFLAGS="$CFLAGS $GDA_CFLAGS $GTK_CFLAGS"
 	AC_CHECK_HEADER(libgda/control-center/gdaui-login-dialog.h,
 			[],
-			[gda_msg="NO.  libgda problem"])
+			[gda_msg="NO.  libgda problem"], [#include <libgda/gda-config.h>])
+	CFLAGS="$OLD_CFLAGS"
 	if test "$gda_msg" = yes; then
 		PKG_CHECK_MODULES(GNOMEDB, [libgnomedb-4.0 >= 3.99.6],
 			[gnomedb_msg="yes"],
diff -p -up gnumeric-1.9.17/plugins/gda/plugin-gda.c.0000 gnumeric-1.9.17/plugins/gda/plugin-gda.c
--- gnumeric-1.9.17/plugins/gda/plugin-gda.c.0000	2009-11-11 01:25:31.000000000 +0100
+++ gnumeric-1.9.17/plugins/gda/plugin-gda.c	2010-01-19 09:25:46.000000000 +0100
@@ -21,8 +21,10 @@
 
 #include <gnumeric-config.h>
 #include <gnumeric.h>
-#include <libgda/libgda.h>
+#include <libgda/gda-config.h>
+#include <libgda/gda-connection.h>
 #include <libgda/control-center/gdaui-login-dialog.h>
+#include <libgnomedb/gnome-db-login.h>
 #include <sql-parser/gda-sql-parser.h>
 #include <string.h>
 

