diff -p -up gnumeric-1.9.17/plugins/gnome-db/plugin-gnomedb.c~ gnumeric-1.9.17/plugins/gnome-db/plugin-gnomedb.c
--- gnumeric-1.9.17/plugins/gnome-db/plugin-gnomedb.c~	2009-09-07 04:03:35.000000000 +0200
+++ gnumeric-1.9.17/plugins/gnome-db/plugin-gnomedb.c	2009-12-16 13:03:48.000000000 +0100
@@ -20,7 +20,7 @@ view_data_sources (GnmAction const *acti
 			_("Could not run GNOME database configuration tool ('%s')"),
 			argv[0]);
 		go_gtk_notice_dialog (wbcg_toplevel (WBC_GTK (wbc)),
-			GTK_MESSAGE_INFO,  msg);
+				      GTK_MESSAGE_INFO,  "%s", msg);
 		g_free (msg);
 	}
 }
